src/entrypoint.sh

```diff
@@ -209,7 +209,7 @@ fi
 
 # ensure the permissions are set correctly
 log "Setting data directory permissions."
-chown -R "${FOUNDRY_UID:-foundry}:${FOUNDRY_GID:-foundry}" /data
+find /data -path "${CHOWN_EXCLUDE_USER_MOUNT:-}" -prune -o -exec chown "${FOUNDRY_UID:-foundry}:${FOUNDRY_GID:-foundry}" {} +
```

Comment:
If we make this a regular expression we can give the user a lot more power.  For example, if you have multiple mounts you could do something like: `/data/Data/readonly\|/data/Data/immutable`.

Let's also change the environment variable name to match the convention of the existing variables.  Since this is modifying the behavior of the startup scripts it should start with `CONTAINER_`.

```suggestion
find /data -regex "${CONTAINER_PRESERVE_OWNER:-}" -prune -o -exec chown "${FOUNDRY_UID:-foundry}:${FOUNDRY_GID:-foundry}" {} +
```

Reply:
I thought I had tried that and it didn't work but, now that I'm doing it again it seems to work. Maybe I had something funky in my regex.

In this example `/data/Data/{foo,bar}` are both directories I want to ignore.
```/data/Data # ls -l /data/Data/
total 0
drwxr-xr-x    2 root     root            64 Dec 13 17:05 bar
drwxr-xr-x   13 root     root           416 Dec 12 04:58 foo
drwxr-xr-x    3 foundry  foundry         96 Dec 13 16:57 modules
drwxr-xr-x    3 foundry  foundry         96 Dec 13 16:57 systems
drwxr-xr-x    3 foundry  foundry         96 Dec 13 16:57 worlds
```

I've set `CONTAINER_PRESERVE_OWNER` to include these directories.
```
/data/Data # echo $CONTAINER_PRESERVE_OWNER
/data/Data/foo\|/data/Data/bar
```

And it appears to work.
```
/data/Data # find /data -regex "${CONTAINER_PRESERVE_OWNER:-}" -prune -o -exec echo {} \;
/data
/data/Config
/data/Config/license.json
/data/Config/options.json
/data/Config/options.json.lock
/data/Data
/data/Data/modules
/data/Data/modules/README.txt
/data/Data/systems
/data/Data/systems/README.txt
/data/Data/worlds
/data/Data/worlds/README.txt
/data/Logs
/data/Logs/debug.log
/data/Logs/error.log```